k8s/upgrade-job/src/upgrade/utils.rs

```diff
@@ -82,6 +86,8 @@ pub(crate) fn all_pods_are_ready(pod_list: ObjectList<Pod>) -> (bool, String, St
                 for condition in *conditions {
                     if condition.type_.eq("Ready") {
                         if condition.status.eq("True") {
+                            let pod_name = pod.name_any();
+                            tracing::info!(pod.name = %pod_name , "Pod is up and running");
```

Comment:
```suggestion
                            tracing::info!(pod.name = %pod_name , "Pod is Ready");
```
Because we are not checking if it is running.